src/ds5/ds5-options.cpp

```diff
@@ -613,7 +613,14 @@ namespace librealsense
         if (_hdr_cfg->is_config_in_process())
             _hdr_option->set(value);
         else
-            _uvc_option->set(value);
+        {
+            if (_hdr_cfg->is_enabled())
+                throw wrong_api_call_sequence_exception(to_string() << "This option cannot be set while HDR is enabled.\n" <<
```

Comment:
Please replace with Log warning. Additionally put the actual control name into the message.
The lines 2 and 3 of the message are not needed and can be removed

Reply:
done